parsons/aws/s3.py

```diff
@@ -346,6 +348,11 @@ def transfer_bucket(self, origin_bucket, origin_key, destination_bucket,
 
             copy_source = {'Bucket': origin_bucket, 'Key': key}
             self.client.copy(copy_source, destination_bucket, dest_key)
+            if remove_original:
+                try:
+                    self.remove_file(origin_bucket, origin_key)
+                except Exception:
+                    pass
```

Comment:
Perhaps we should include some logging to indicate that it failed, or lift up the boto3 error that was raised in the logs.